rviz_common/src/rviz_common/visualization_manager.cpp

```diff
@@ -437,8 +437,7 @@ void VisualizationManager::onUpdate()
   }
 
   frame_count_++;
-  constexpr std::chrono::milliseconds ten_ms = std::chrono::milliseconds(10);
-  if (render_requested_ || wall_dt > static_cast<uint64_t>(ten_ms.count())) {
+  if (render_requested_ || wall_dt > static_cast<uint64_t>(wall_diff.count())) {
```

Comment:
This doesn't make sense to me... `wall_dt` should be the same value as `wall_diff`... 🤔

Reply:
🤦‍♂ I'm sorry @wjwwood I don't know why I thought this was the correct comparison to make...

I need to compare `wall_diff` with `std::chrono::milliseconds(10)`. Thanks for catching that!

Reply:
`wall_diff > std::chrono::milliseconds(10)` both compiles and makes sense (to me at least). Let me know if there's any more adjustments.